controllers/front/ProductController.php

```diff
@@ -961,6 +961,10 @@ private function getIdProductAttribute($useGroups = false)
                     );
                 } catch (PrestaShopObjectNotFoundException $e) {
                     // Don't care just set $requestedIdProductAttribute to null
+                    PrestaShopLogger::addLog(
+                        __METHOD__ . ' - Id Product Attribute not found. ' . print_r([$this->product, $groups], true)
```

Comment:
I'm curious about it, how does this log message render in BO Logs page?

Reply:
Hahaha, too tired today, thinking it was a file logger... I change this now =)